media/img/firefox/whatsnew/whatsnew_background.svg

```diff
@@ -1 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1313 612.51"><defs><clipPath id="a"><path fill="none" d="M-.13.97h1313v1036H-.13z"/></clipPath><clipPath id="b"><path d="M601.6-4.12C755.81 93 815.12 298.07 734.07 453.91s-271.76 203.43-426 106.31L-898.64-199.85c-154.21-97.13-213.52-302.19-132.47-458s271.75-203.43 426-106.31z" fill="none"/></clipPath><linearGradient id="c" x1="278.53" y1="188.44" x2="317.62" y2="735.77" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#201135"/><stop offset="1" stop-color="#371868"/></linearGradient><clipPath id="d"><path d="M1577.25-1247c108.94-136.49 319.06-162.34 469.33-57.75s183.76 300 74.82 436.52l-964.48 1208.4C1048 476.65 837.86 502.51 687.6 397.92l-1330.38-926c-150.27-104.6-183.77-300-74.84-436.52s319.06-162.35 469.33-57.75L810-285.73z" fill="none"/></clipPath><linearGradient id="e" x1="762.09" y1="-59.33" x2="617.44" y2="695.2" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#20123a"/><stop offset="1" stop-color="#4b2292"/></linearGradient><clipPath id="f"><path d="M966.42-1281.06c71.87-159.16 269.11-236.08 440.54-171.82s252.14 245.39 180.26 404.54L950.89 360.74C879 519.89 681.78 596.82 510.35 532.55L-1007.46-36.4c-171.43-64.26-252.14-245.38-180.27-404.54S-918.62-677-747.19-612.76l1207.41 452.6z" fill="none"/></clipPath><linearGradient id="g" x1="441.26" y1="112.54" x2="363.07" y2="607.74" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#20123a"/><stop offset="1" stop-color="#332869"/></linearGradient></defs><g clip-path="url(#a)"><g clip-path="url(#b)"><path fill="url(#c)" d="M1463.6 148l-630.87 960.61L-938.08-212.73l630.86-960.61L1463.6 148z"/></g><g clip-path="url(#d)"><path fill="url(#e)" d="M680.24-2015L2388 617.22 881.76 1699.07-826.01-933.15 680.24-2015z"/></g><g clip-path="url(#f)"><path fill="url(#g)" d="M-92.43-1803.87l2304.7 2129.18-1192.57 1420.2L-1285-383.67l1192.57-1420.2z"/></g></g></svg>
```

Comment:
**Issue:** this image appears to still be used in the Nightly WNP? http://localhost:8000/en-US/firefox/128.0a1/whatsnew/